os/arch/arm/src/imxrt/imxrt_norflash.c

```diff
@@ -79,6 +79,7 @@
 #include "imxrt_clock.h"
 #include "imxrt_iomuxc.h"
 #include "chip/imxrt_memorymap.h"
+#include "imxrt_log.h"
```

Comment:
If this nor flash accepted all of below conditions, you can use progmem interface.
1. nor flash is your product
2. It should be used only on the imxrt board.
3. All of your imxrt board use same spi bus for nor flash.